### PR TITLE
docs: Add Oasis CLI commands to stake requirements

### PR DESCRIPTION
**Pull request by matevz**
Part of #978 

- Adds Oasis CLI commands to stake requirements chapter
- Adds ROFL-specific thresholds
- Revamps the Staking thresholds chapter into one big table with footnotes

Merge after https://github.com/oasisprotocol/cli/pull/318

[PREVIEW](https://deploy-preview-1022--oasisprotocol-docs.netlify.app/node/run-your-node/prerequisites/stake-requirements)

### CODE REVIEW

File: docs/node/run-your-node/prerequisites/stake-requirements.md [hunk 1]
```diff
@@ -1,38 +1,49 @@
 # Stake Requirements
 
-This page provides an overview of the stake requirements to become a validator
-on the consensus layer, to run a ParaTime compute nodes and to run observer
-nodes on the Oasis Network.
+This page provides the current stake requirements to first register your account
+as an entity on the consensus layer, to become a validator and finally to run a
+ParaTime or the observer compute node.
```
Comment (kostko) [inline docs/node/run-your-node/prerequisites/stake-requirements.md]:
```suggestion
ParaTime compute node or an observer node.
```

File: docs/node/run-your-node/prerequisites/stake-requirements.md [hunk 2]
```diff
@@ -44,39 +55,56 @@ Foundation, see the [Delegation Policy] document.
...
- currently is:
+To run a ParaTime compute node you will need to satisfy the following stake
+requirements:
 
 |          | Mainnet        | Testnet  |
 | -------- | -------------- | -------- |
 | Sapphire | 5,000,100 ROSE | 100 TEST |
 | Emerald  | 5,000,100 ROSE | 100 TEST |
 | Cipher   | 100 ROSE       | 100 TEST |
+| Must be in the validator set | Yes | No |
```
Comment (kostko) [inline docs/node/run-your-node/prerequisites/stake-requirements.md]: Please see the original PR why I said that this is a bad idea. The requirement is per-paratime, not global, so it shouldn't be specified as a row in this table.

File: docs/node/run-your-node/prerequisites/stake-requirements.md [hunk 3]
```diff
@@ -44,39 +55,56 @@ Foundation, see the [Delegation Policy] document.
...
-| Emerald  | Yes                     | No                      |
-| Cipher   | Yes                     | No                      |
+[`oasis account show <paratime id>`]: ../../../general/manage-tokens/cli/network.md#show-id
 
 ## ParaTime Observer Node
 
 To run a ParaTime observer nodes, you will need to register an entity and add
 your observer nodes to the entity descriptor.
+
+## ParaTime-specific Application Thresholds
```
Comment (kostko) [inline docs/node/run-your-node/prerequisites/stake-requirements.md]: Important note that this balance needs to exist _inside tha paratime_, while others exist in the consensus layer and need to be delegated.

File: docs/node/run-your-node/prerequisites/stake-requirements.md [hunk 4]
```diff
@@ -1,38 +1,49 @@
...
+top entities by stake to be included in the **validator set**. You will also
+need to register a node, if you want to run an **observer node**.
 
-You can use the Oasis CLI to check your account and the network parameters. Use
-the [`oasis account show`] command to check your entity's account and
-[`oasis network show native-token`] to see the current values used by the network.
+|                           | Mainnet  | Testnet  |
+|---------------------------|----------|----------|
+| Each node registration    | 100 ROSE | 100 TEST |
+| Size of the validator set | 120      | 110      |
```
Comment (amela) [inline docs/node/run-your-node/prerequisites/stake-requirements.md]: Don't you need +100 tokens to register an entity? I would also remove the validator set size from this table, or make another table or sth for validator set size.

Comment (matevz) [reply docs/node/run-your-node/prerequisites/stake-requirements.md]: I revamped the whole chapter. Please take a look.

**Review by kostko — APPROVED** [review]
Nice!

**Pull request merged**